packages/bpk-component-split-input/src/BpkInputField.js

```diff
@@ -62,6 +63,7 @@ BpkInputField.propTypes = {
   value: PropTypes.string,
   focus: PropTypes.bool.isRequired,
   index: PropTypes.number.isRequired,
+  name: PropTypes.string.isRequired,
```

Comment:
We probably don't want to make it a required field.. as it would be a breaking change and a major release. The component can still work without a name attribute but we want to allow consumers to define if needed.

Reply:
Makes sense!